docs/config.md

```diff
@@ -20,11 +20,11 @@ name = "YOUR NAME"
 For a more complicated example,
 
 ```toml
-[template-aliases]
-"format_short_id(id)" = "id.shortest(12)"
+[colors]
+"commit_id prefix" = { bold = true}
```

Comment:
I'd write "For more complicated examples" and let both examples, as the `template-aliases` example illustrates defining a function. Both are useful to me.

Reply:
Interesting idea. I'm wondering if this means I should rearrange things a little, perhaps extract this to a separate subsection.

Reply:
I added both examples and rewrote things a little.